webapp/app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -1085,9 +1094,10 @@ msgstr "Registrieren und persönlichen Freischaltcode beantragen"
 msgid "form.unlock-code-request.input-intro"
 msgstr ""
 "Mit Ihrer Registrierung beantragen Sie einen Freischaltcode bei Ihrer "
-"Finanzverwaltung. Sie erhalten diesen mit einem Brief wenige Tage nach "
-"erfolgreicher Beantragung. Wenn Sie die Zusammenveranlagung nutzen "
-"möchten, muss sich nur eine Person registrieren."
+"Finanzverwaltung. Sie erhalten diesen mit einem Brief <b>innerhalb von "
+"zwei Wochen</b> nach erfolgreicher Beantragung. Wenn Sie die "
+"Zusammenveranlagung nutzen möchten, muss sich nur eine Person "
+"registrieren."
```

Comment:
This should probably not be here anymore, so I think you can just rerun the babel script to get rid of it?

Reply:
Fixed that